bundle/manifests/netobserv-operator.clusterserviceversion.yaml

```diff
@@ -1070,7 +1072,7 @@ spec:
                 - name: RELATED_IMAGE_CONSOLE_PLUGIN
                   value: quay.io/netobserv/network-observability-console-plugin:v1.9.2-community
                 - name: RELATED_IMAGE_CONSOLE_PLUGIN_COMPAT
-                  value: quay.io/netobserv/network-observability-console-plugin-pf4:v1.8.2-community
+                  value: quay.io/netobserv/network-observability-console-plugin-pf4:v1.9.1-community
```

Comment:
Why is there `network-observability-console-plugin:v1.9.2-community` alongside `network-observability-console-plugin-pf4:v1.9.1-community` here ? :thinking:

Reply:
yeah, some wrong things happening with the `make update-bundle` I think, I'll remove that

Reply:
yeah we should fix that in parallel